ts/features/messages/components/MessageDetail/MessageDetailsReminderExpired.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
```

Comment:
why not merge `expired` and `expiring` with a mapping function to derive the props for `alert`, to then merge them?